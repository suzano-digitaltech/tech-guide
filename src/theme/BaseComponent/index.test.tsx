import React from "react";
import { render } from "@src/infra/tests/react-testing-library";
import BaseComponent from "./index";

describe("<BaseComponent />", () => {
  it("renders as expected", () => {
    const { container } = render(<BaseComponent />);
    expect(container).toMatchSnapshot();
  });
});
