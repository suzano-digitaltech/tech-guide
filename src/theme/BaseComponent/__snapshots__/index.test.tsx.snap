// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseComponent /> renders as expected 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

<div>
  <div
    class="c0 r0"
  />
</div>
`;
